include/deal.II/lac/utilities.h

```diff
@@ -147,6 +168,73 @@ namespace Utilities
       return diagonal[k-1] + f->l2_norm();
     }
 
+
+    template <typename OperatorType, typename VectorType>
+    void Chebyshev_filter(VectorType &x,
+                          const OperatorType &op,
+                          const unsigned int degree,
+                          const double a,
+                          const double b,
+                          const double a_L,
+                          VectorMemory<VectorType> &vector_memory)
+    {
+      Assert (degree > 0,
+              ExcNotImplemented());
+
+      const bool scale = !numbers::is_nan(a_L);
+      Assert (a < b,
+              ExcMessage("Lower bound of the unwanted spectrum should be smaller than the highest bound."));
+
+      Assert (a_L <= a || a_L >= b || !scale,
+              ExcMessage("Scaling point should be outside of the filtering range."));
```

Comment:
filtering range -> unwanted spectrum
(so as to use the same terminology as in the documentation of the class).